pyquil_for_azure_quantum/__init__.py

```diff
@@ -182,7 +154,7 @@ def get_qvm() -> AzureQuantumComputer:
     Raises:
         KeyError: If required environment variables are not set.
     """
-    return AzureQuantumComputer(target="rigetti.sim.qvm", qpu_name="qvm")
+    return AzureQuantumComputer(target=RigettiTarget.QVM.value, qpu_name="qvm")
```

Comment:
Do we need to update https://github.com/microsoft/qdk-python/blob/main/azure-quantum/azure/quantum/target/rigetti/target.py#L151 as well?

Reply:
Sorry I'm missing it, but I'm not connecting the dots here. What should we update?